Doc/library/math.rst

```diff
@@ -199,8 +199,8 @@ Power and logarithmic functions
 
 .. function:: exp(x)
 
-   Return ``e**x``.
-
+   Return e raised to the power *x*, where e = 2.718281... is the base
```

Comment:
`e = 2.718281...` is not a code. I don't think it should have the code markup.

Emphasized `*e*` is used in descriptions of `log()` and `log1p()`. But it also is used in `frexp()`. I suggest to rename `e` in `frexp()` description for avoiding confusion.